lib/Jsrt/Jsrt.cpp

```diff
@@ -1332,6 +1689,23 @@ CHAKRA_API JsGetTypedArrayInfo(_In_ JsValueRef typedArray, _Out_opt_ JsTypedArra
             *byteLength = typedArrayBase->GetByteLength();
         }
     }
+
+#if ENABLE_TTD
```

Comment:
In JsCreateTypedArray we have PERFORM_JSRT_TTD_RECORD_ACTION_NOT_IMPLEMENTED then why do we need these? Isn't the scenario involving TypedArray broken?